practice-two/src/index.html

```diff
@@ -32,6 +32,71 @@ <h1 class="logo-text">Travlog</h1>
           /></a>
         </nav>
       </header>
+      <main>
+        <section class="best-location-section">
+          <span class="section-title best-location-title">
+            Top Destination
+          </span>
+          <h2 class="section-heading best-location-heading">
+            Explore top destination
+          </h2>
+          <div class="carousel-button">
+            <button class="previous-btn">
+              <img
+                src="assets/images/previous-icon.png"
+                width="15px"
+                height="15px"
```

Comment:
We should use CSS styling for easier maintenance

Reply:
i will fix it!